NiL.JS/ExportTable.cs

```diff
@@ -23,12 +23,52 @@ public sealed class ExportTable : IEnumerable<KeyValuePair<string, JSValue>>
 
                 return result;
             }
-            internal set
+             set
             {
                 _items[key] = value;
             }
         }
+        /// <summary>
+        /// Add object constructor to export 
+        /// </summary>
+        /// <remarks>We dont recommend to use, because api can change and its can be broken in future. Use indexers</remarks>
+        /// <param name="type">Class</param>
+        /// <param name="name"></param>
+        public void DefineConstructor(Type type, string name)
+        {
+            var ctor = GlobalContext.CurrentGlobalContext.GetConstructor(type);
+            _items.Add(name, ctor);
+            ctor._attributes |= JSValueAttributesInternal.DoNotEnumerate;
```

Comment:
this line looks senseless

Reply:
Why seems senseless?

Reply:
There is no way to enumerate `ExportTable`